tests/FakeItEasy.Specs/FakeItEasy.Specs.ruleset

```diff
@@ -5,4 +5,4 @@
     <Rule Id="CA1003" Action="None" />
     <Rule Id="CA1009" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
Surprising, given no other changes to the file. I'm not that worked up either way. I'm sure you either opened the text view and editorconfig changed things, or you _didn't_ open the text view and regular VS changed things.